build/src/base/event/breadcrumb.d.ts

```diff
@@ -0,0 +1,40 @@
+import { Json } from '../../utils';
+/**
+ * Breadcrumb severity level
+ */
+export declare type BreadcrumbLevel = 'fatal' | 'error' | 'warning' | 'info' | 'debug';
+/**
+ * Breadcrumb type - controls categorization and UI appearance
+ * Common types suitable for both client and backend
+ */
+export declare type BreadcrumbType = 'default' | 'request' | 'ui' | 'navigation' | 'logic' | 'error';
+/**
+ * Single breadcrumb entry - represents an event that occurred before the error
+ */
+export interface Breadcrumb {
+    /**
+     * Timestamp when the breadcrumb occurred
+     * ISO 8601 string or Unix timestamp (seconds or milliseconds)
+     */
+    timestamp: string | number;
```

Comment:
decide how we will store timestamp: via string or number. And explain it in jsdoc